infrastructure/serve/terragrunt.hcl

```diff
@@ -18,6 +18,16 @@ include "root" {
 
 dependency "core" {
   config_path = "../core"
+
+  mock_outputs = {
+    core_rg_name                = "core_rg_name"
```

Comment:
more of a question than a comment....

when defining the value of mock outputs, is there any value in adding the work `mock` to the value?

e.g. 
``` hcl
core_rg_name                = "mock_core_rg_name"
```

Would this help if we accidentally ended up with a mock output being used for something unexpected.